### PR TITLE
advanced access page

### PR DESCRIPTION
**Pull request by tyleretters**
this pr supersedes #337

adds an advanced access page, merges in the sftp instructions, updates smb instructions, fixes typo on the password section

Screen Shot 2022-01-17 at 11 05 49

open to feedback!

### CODE REVIEW

File: norns/wifi-files.md [hunk 1]
```diff
@@ -77,16 +77,16 @@ Here's the `dust` file tree:
...
-    tape/             -- tape recordings
-    ...
-  code/           -- contains scripts and engines
-    awake/
-    mlr/
-    ...
-    we/
-  data/           -- contains user data created by scripts
-    awake/            -- for example, pset data
+audio/-- audio files
```
Comment (tehn) [inline norns/wifi-files.md]: this whitespace was actually for legibility, can you restore it?

**Comment by tehn** [comment]
looks great, thank you!

File: norns/wifi-files.md (hunk 1 above)
Comment (tyleretters) [reply norns/wifi-files.md]: oh weird. must've been an IDE or copy paste issue. yes i'll fix.

**Comment by tyleretters** [comment]
fixed the whitespace. tested locally. looks good to merge.

**Review by tehn — APPROVED** [review]

**Comment by tyleretters** [comment]
@tehn how do we get the lower table of contents off the page? i don't see where it is being generated from:
Screen Shot 2022-01-17 at 14 16 45

**Pull request merged**